client/modules/User/components/Collection.jsx

```diff
@@ -69,22 +70,23 @@ const ShareURL = ({ value }) => {
 
 ShareURL.propTypes = {
   value: PropTypes.string.isRequired,
+  t: PropTypes.func.isRequired
 };
 
 const CollectionItemRowBase = ({
-  collection, item, isOwner, removeFromCollection
+  collection, item, isOwner, removeFromCollection, t
 }) => {
   const projectIsDeleted = item.isDeleted;
 
   const handleSketchRemove = () => {
     const name = projectIsDeleted ? 'deleted sketch' : item.project.name;
 
-    if (window.confirm(`Are you sure you want to remove "${name}" from this collection?`)) {
+    if (window.confirm(t('Collection.DeleteFromCollection', { name_sketch: name }))) {
       removeFromCollection(collection.id, item.projectId);
     }
   };
 
-  const name = projectIsDeleted ? <span>Sketch was deleted</span> : (
+  const name = projectIsDeleted ? <span>{t('Collection.SketchDeletedSpan')}</span> : (
```

Comment:
Why does this have `Span` in the translation key name? Perhaps is could be `Collection.SketchDeleted`?

Reply:
Mistake, corrected